trace/trace_test.go

```diff
@@ -242,6 +245,13 @@ func TestSpan(t *testing.T) {
 
 	span.AddField("f1", "v1")
 	assert.Equal(t, "v1", span.ev.Fields()["f1"].(string), "after adding a field, field should exist on the span")
+	// add an error
+	expected := errors.New("test error")
+	span.AddField("error", expected)
+	assert.Contains(t, span.ev.Fields(), "error")
+	msg, ok := span.ev.Fields()["error"].(string)
+	require.True(t, ok)
```

Comment:
I'd prefer to use the existing assert terminology:
```suggestion
	assert.Equal(t, true, ok)
```